notifier.go

```diff
@@ -45,7 +49,10 @@ func (n *SlackNotifier) Approved(c context.Context, repo *github.Repository, pr
 		prLink(review.HTMLURL(), repo, pr),
 	)
 
-	return n.notifyChannel(c, notificationChannel, text)
+	var messageBlocks []slackapi.Block
+	messageBlocks = append(messageBlocks, buildTextMessageBlock(text))
+
+	return n.notifyChannel(c, notificationChannel, messageBlocks)
```

Comment:
the initial text bit can all be like this, so perhaps `buildTextMessageBlock()` should return a slice?
```suggestion
	blocks := []slackapi.Block{buildTextMessageBlock(text)}
	return n.notifyChannel(c, notificationChannel, blocks)
```